monai/networks/nets/senet.py

```diff
@@ -176,6 +183,65 @@ def __init__(
             elif isinstance(m, nn.Linear):
                 nn.init.constant_(torch.as_tensor(m.bias), 0)
 
+        if pretrained:
+            self._load_state_dict(pretrained_arch, progress)
+
+    def _load_state_dict(self, arch, progress):
+        """
+        This function is used to load pretrained models.
+        """
+        model_urls = {
+            "senet154": "http://data.lip6.fr/cadene/pretrainedmodels/senet154-c7b49a05.pth",
+            "se_resnet50": "http://data.lip6.fr/cadene/pretrainedmodels/se_resnet50-ce0d4300.pth",
+            "se_resnet101": "http://data.lip6.fr/cadene/pretrainedmodels/se_resnet101-7e38fcc6.pth",
+            "se_resnet152": "http://data.lip6.fr/cadene/pretrainedmodels/se_resnet152-d17c99b7.pth",
+            "se_resnext50_32x4d": "http://data.lip6.fr/cadene/pretrainedmodels/se_resnext50_32x4d-a260b3a4.pth",
+            "se_resnext101_32x4d": "http://data.lip6.fr/cadene/pretrainedmodels/se_resnext101_32x4d-3b2fe3d8.pth",
+        }
+        if arch in model_urls.keys():
+            model_url = model_urls[arch]
+        else:
+            error_msg = (
+                "only senet154, se_resnet50, se_resnet101,  se_resnet152, se_resnext50_32x4d "
+                + "and se_resnext101_32x4d are supported to load pretrained weights."
+            )
+            raise AssertionError(error_msg)
```

Comment:
suggest to use:
```py
raise ValueError(
    "only senet154, se_resnet50, se_resnet101,  se_resnet152, se_resnext50_32x4d, \
    and se_resnext101_32x4d are supported to load pretrained weights."
)
```

Reply:
Hi Nic, I just found that if using this kind of line breaks, the printed results will introduce extra spaces. During my latest PR (#1914) I fixed it.

Reply:
Hi @ericspod ,

Could you please help share some comments here? Maybe I misunderstood your previous comments in my PR? You suggested this print: https://github.com/Project-MONAI/MONAI/blob/master/monai/transforms/io/array.py#L147

Thanks.

Reply:
This will replicates the issue since the indentation gets included with the string:

```python
if True:
    s="Hello \
    world"
print(s)  # prints "Hello     world"
```

However this will work as expected, the important difference is to close the strings off with " so the newline escape becomes essentially an instruction to concat strings together:

```python
if True:
    s="Hello "\
    "world"
print(s)  # prints "Hello world"
```

Reply:
I see, thanks.